packages/neuron-ui/src/components/SUDTSend/index.tsx

```diff
@@ -405,6 +414,12 @@ const SUDTSend = () => {
                       <TooltipIcon width={12} height={12} />
                     </span>
                   ) : null}
+                  {v.key === SendType.secp256Cheque ? (
```

Comment:
How about showing this warning message only on testnet, because nobody uses it on mainnet

Reply:
I think it's ok now, we can only show this and the lock-time warning on `testnet`. Because currently we only support light client testnet.